src/backend/distributed/transaction/remote_transaction.c

```diff
@@ -225,8 +226,7 @@ StartRemoteTransactionCommit(MultiConnection *connection)
 
 		if (!SendRemoteCommand(connection, command.data))
 		{
-			ReportConnectionError(connection, WARNING);
-			MarkRemoteTransactionFailed(connection, dontRaiseError);
+			HandleRemoteTransactionConnectionError(connection, dontRaiseError);
```

Comment:
`dontRaiseError` is confusing, given all the other uses use `raiseError`; can you make it consistent? I'm not a huge fan of this `const bool` stuff anyhow (IMO everyone should be using an editor capable of showing them the function declaration), but if we must, making them positive when true and negative when false will add more confusion.

Reply:
Yep, the double negative bugged me too. Fixing.